packages/allure-cucumberjs/test/specs/allure_cucumberjs_test.ts

```diff
@@ -396,4 +417,29 @@ describe("CucumberJSAllureReporter", () => {
     expect(severity?.value).eq("bar");
     expect(tags).length(1);
   });
+
+
+  it("should add encoding type to images", async () => {
```

Comment:
as far as I can see test name is wrong

Reply:
That whole test was wrong.  I should have deleted it initially as it was a duplicate of the other test.